src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -532,7 +532,7 @@ For details on valid range queries, please refer to Search Matching section in C
     <comment>{0} is the DICOM tag name.</comment>
   </data>
   <data name="RetrieveServiceFileTooBig" xml:space="preserve">
-    <value>Requested DICOM instance size is above the supported limit of {0}. The entire instance can be retrieved with the original transfer syntax by specifying 'transfer-syntax=*' in the Accept header.</value>
+    <value>Requested DICOM instance size is above the supported limit of {0}. The entire instance can be retrieved with the original transfer syntax by specifying 'transfer-syntax=*' in the Accept header instead of attempting to transcode or render.</value>
```

Comment:
I still believe this message is misleading for user trying to call render endpoint. May be you can add a variable to differentiate the error message and be specific to render instance/frame